clio-client/src/test/scala/org/broadinstitute/clio/client/dispatch/DeliverArraysExecutorSpec.scala

```diff
@@ -30,9 +30,9 @@ class DeliverArraysExecutorSpec extends BaseClientSpec with AsyncMockFactory {
       Some(URI.create(s"gs://bucket/the-vcf${ArraysExtensions.VcfGzTbiExtension}")),
     gtcPath = Some(URI.create(s"gs://bucket/the-gct${ArraysExtensions.GtcExtension}")),
     grnIdatPath =
-      Some(URI.create(s"gs://bucket/the-grn${ArraysExtensions.IdatExtension}")),
+      Some(URI.create(s"gs://bucket/the-grn${ArraysExtensions.GrnIdatExtension}")),
```

Comment:
Not sure what the point of this change is, was this required to get the tests to pass?

Reply:
You'll need to change the logic in the `DeliverArraysExecutor` which currently uses the generic idat extension for this to have an effect on main code.

Reply:
check.